### PR TITLE
base: render_record_authors macro fix

### PR DESCRIPTION
**Pull request by kaplun**
* Fixes render_record_authors() macro to correctly handle records
  with missing authors.

Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

**Comment by kaplun** [comment]
@jalavik, @jmartinm this is against master-elasticsearch.

**Comment by jalavik** [comment]
:shipit:

File: inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl [hunk 1]
```diff
@@ -18,16 +18,16 @@
 #}
 
 {% macro render_record_authors(number_of_displayed_authors) %}
-  {% if record.get('authors')|length > 0 %}
+  {% if record.get('authors', [])|length > 0 %}
```
Comment (jmartinm) [inline inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl]: we can move to the more simple syntax:

```python
{% if record.authors | length > 0 %}
```

if the key does not exist jinja2 does not crash and `length` returns 0.

Comment (jalavik) [inline inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl]: Ah nice!

**Comment by jmartinm** [comment]
do not :shipit: :P

**Comment by jmartinm** [comment]
What was the latest status of this?

**Comment by kaplun** [comment]
That I have to fix it. I'll do it ASAP

**Comment by kaplun** [comment]
OK. What about now?

**Comment by jmartinm** [comment]
Looks good to me. :shipit: ;)

**Pull request merged**